content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -231,6 +233,37 @@ when both the following statements apply:
 When a Pod's containers are Ready but at least one custom condition is missing or
 `False`, the kubelet sets the Pod's [condition](#pod-conditions) to `ContainersReady`.
 
+### Pod network readiness {#pod-has-network}
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
```

Comment:
Do folks need to enable a feature gate for this to work? (yes)

Please mention `PodHasNetworkCondition` and link to https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/

To find existing examples of use, search the English docs for `state="alpha"`.

Reply:
done